client/src/components/Homepage/tickets-home-draft.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { Fragment } from 'react';
+import Navbar from '../navbar/Navbar';
+import './style.css';
+import MainSidebar from '../MainSidebar';
+
+export default () => (
```

Comment:
@Anies12 It's not a necessity, but if you give these exports names (for example `const HomeDraft = () => ..., export default HomeDraft` then you'll see those names in react devtools instead of seeing `<Unknown>`.  This can make debugging easier.